performance.lic

```diff
@@ -0,0 +1,156 @@
+custom_require.call(%w(common common-travel))
+  
+class Performance
+  
+  include DRC
+  include DRCT
+  
+  def initialize
+    no_pause_all
+    startTime = Time.new.to_i
+
+    @settings = get_settings
+    @song_list = get_data('perform').perform_options
+    UserVars.song = @song_list.first.first unless UserVars.song
+    fput('release ecry') if DRStats.bard?
+
+    instrument = @settings.instrument
+
+    hand_armor = @settings.hand_armor
+    if instrument != nil
+      bput("get my #{instrument}","You get")
+      bput("remove my #{hand_armor}","You pull off","Remove what")
+      bput("stow my #{hand_armor}","You put","Stow what")
+    end
+    
+    if play_song
+      loop do 
+        runTime = Time.new.to_i - startTime
+        break if runTime > 600
+        break if DRSkill.getxp('Performance') > 30
```

Comment:
💡 The `DRSkill.getxp('Performance') > 30` pattern is repeated three times. What do you think about extracting it to a helper function?